src/main/java/uk/gov/companieshouse/api/accounts/configuration/AmazonS3Configuration.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.companieshouse.api.accounts.configuration;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.Protocol;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AmazonS3Configuration {
+
+    @Value("${aws.configs.cloud.host}")
+    private String cloudProxyHost;
+
+    @Value("${aws.configs.cloud.port}")
+    private Integer cloudProxyPort;
```

Comment:
(Very minor) I find the use of `cloud` in these variable names is unnecessary? And possibly in the properties keys?